shared/wallets/wallet/container.js

```diff
@@ -51,18 +68,7 @@ const mergeProps = (stateProps, dispatchProps) => {
   }
 }
 
-const paymentsFromState = memoize((paymentsMap: ?I.Map<Types.PaymentID, Types.Payment>) => {
-  if (!paymentsMap) {
-    return ['notLoadedYet']
-  }
-  if (paymentsMap.size === 0) {
-    return ['noPayments']
-  }
-  const payments = paymentsMap.toList()
-  return payments // payments always have a time
-    .sort((p1, p2) => (p2.time && p1.time && p2.time - p1.time) || 0)
-    .map(p => ({paymentID: p.id, status: p.statusSimplified}))
-    .toArray()
-})
+const sortAndStripTimestamps = (p: Array<{paymentID: Types.PaymentID, timestamp: number}>) =>
```

Comment:
we ok to lose the memoize?

Reply:
I think so, I don't notice any perf difference with it gone and we can always add it back in if we need it